native/themes/colors.js

```diff
@@ -215,11 +215,16 @@ function useOverlayStyles<IS: Styles>(obj: IS): StyleSheetOf<IS> {
   ]);
 }
 
+function markdownStyle<IS: Styles>(obj: IS, theme: string): StyleSheetOf<IS> {
```

Comment:
1. `theme` should be type `GlobalTheme`
2. Let's call this function `getStylesForTheme`